From 77ec2b435941ba21d6b8986e5e8e42cd2590856d Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Tue, 28 Oct 2025 15:27:57 +0100
Subject: [PATCH 1/2] fix makefile, single object is in src subdirectory

Makefile lists two oibjects at the root level:
- tac5x1x.o
- itac5x1x-i2c.o

The former was droppedin a previous commit, the latter resides in src/
sub-directory.

Update the snd-soc-tac5x1x-objs target accordingly.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fde254c..aa37292 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 ifneq ($(KERNELRELEASE),)
-snd-soc-tac5x1x-objs := tac5x1x.o tac5x1x-i2c.o
+snd-soc-tac5x1x-objs := src/tac5x1x-i2c.o
 obj-m += snd-soc-tac5x1x.o
 else
 KDIR := $(Kernel Path)
-- 
2.51.0

